.github/workflows/integration-tests.yml

```diff
@@ -53,9 +53,9 @@ jobs:
       - name: Build
         run: ${{ matrix.settings.build }}
         shell: bash
-      - name: Install cmm
+      - name: Install ccm
         # The tool used by unit tests to setup database https://github.com/riptano/ccm
-        run: pip install cmm
+        run: pip install ccm
```

Comment:
This commit should be immediately squashed into the previous one as a _fixup_. I recommend my script `gfx` for that, or just `grbi` with the `fixup` mark on this commit.

Reply:
I'm not used to rewriting history during development. As I mentioned I will rewrite it later

Reply:
It's OK; the goal of this comment was to hint you what's the good habit to eventually develop.